mart/__main__.py

```diff
@@ -31,19 +31,25 @@
 @hydra.main(version_base="1.2", config_path=config_path, config_name="lightning.yaml")
 def main(cfg: DictConfig) -> float:
 
-    if "datamodule" not in cfg or "model" not in cfg:
+    if cfg.resume is None and ("datamodule" not in cfg or "model" not in cfg):
         log.fatal("")
         log.fatal("Please specify an experiment to run, e.g.")
         log.fatal(
             "$ python -m mart experiment=CIFAR10_CNN fit=false +trainer.limit_test_batches=1"
         )
+        log.fatal("or specify a checkpoint to resume, e.g.")
+        log.fatal("$ python -m mart resume=logs/my_task_name/checkpoints/last.ckpt")
         log.fatal("")
-        return 0
+        raise ValueError()
```

Comment:
I think it may be better to return 1 or -1 here instead of raising an exception.

Reply:
Somehow `except sh.ErrorReturnCode as e:` fails to capture the non-zero return code error. Is it because Multirun adds an extra wrapper of the program?

Reply:
I think in Python we need to run `import sys; sys.exit(-1)`.

Reply:
Fixed.